### PR TITLE
[Community] Community Managers page

### LINKED ISSUE #2484: Addition of Community Managers section
**Current Behavior**

**Desired Behavior**

- Remove "Community" in Our Community MeshmMates.
- Add a community managers section under Meshmates with profiles.

---
**Screenshots / Mockups**

**Alternatives**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**

- Removed Community from "Community MeshMates" title.
- Added a new entry of section under Community > Members >  Community Managers(under MesMates).
- Updated the gatsby graphql query for managers.
- New page for Layer5 Community Managers.
- Fix other minor lint errors for some files.

This PR fixes #2484 

**Notes for Reviewers**
Page and query for managers tested OK

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 36b7f2484ca56bf76d9bb4368daf74431e3b9519 at: https://61e40ca6232a13ae711873c1--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@Chadha93 could we discuss this on the websites' call tomorrow if you're available.

**Comment by Chadha93** [comment]
> @Chadha93 could we discuss this on the websites' call tomorrow if you're available.?

Sure @warunicorn19 sounds good.

**Comment by warunicorn19** [comment]
Thank you @Chadha93

**Comment by leecalcote** [comment]
remove mega menu item

**Comment by l5io** [comment]
🚀 Preview for commit d317eb1c109e9334fb26334babf9371a6d712777 at: https://61e4dc39a71e1ebdc073b330--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7c5dab2cdb5bd99080b415445d7b44dca0236b9c at: https://61e57b75a71e1e793f73af46--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 208b31a1c02dd0f915a4048d9351835e851a1938 at: https://61e714e3acb7ff139ab77fa1--layer5ng.netlify.app

**Comment by Chadha93** [comment]
@warunicorn19 Updated according to the suggestions made on the call, let's review and tie this off. :)

File: src/sections/Community/CommunityManagers/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          <Row>
+            {data.allMdx.nodes.map(({ id, frontmatter, fields }) => (
+              <Col xs={12} sm={6} lg={4} key={id}>
+                <ProfileCard frontmatter={frontmatter} cardlink = {fields.slug}/>
+              </Col>
+            ))}
+          </Row>
+        </div>
+        <div className="expect">
+          <h2> What Does the Community Managers Do? </h2>
```
Comment (warunicorn19) [inline src/sections/Community/CommunityManagers/index.js:60]:
```suggestion
          <h2> What Does a Community Manager Do? </h2>
```

File: src/sections/Community/CommunityManagers/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          <p> Layer5 Community Managers generally oversee community activities and support ongoing initiatives. They are responsible for the health and growth of the community.
+             Community Managers cultivate an environment which attracts new community members by ensuring that timely and completed responses are provided to questions asked. 
+             Aspects of marketing including member and project promotion as well as aspects of project management by organizing meetings and triaging issues are under the purview of Community Managers. </p>
+          <p>Community Managers work to ensure that existing community members are retained by helping those members stay engaged in projects on an ongoing basis.</p>
+        </div>
+        <div className="manager-meet">
+          <h2> Want To Know More About Community Managers?</h2>
+          <p>Checkout their Community roles and responsibilites on <a href="https://layer5.io/community/handbook/community-roles">Community Handbook</a></p>
```
Comment (warunicorn19) [inline src/sections/Community/CommunityManagers/index.js:68]:
```suggestion
          <p>Take a look at the <Link to="/community/handbook">Community Handbook</a></p>
```

File: src/sections/Community/CommunityManagers/index.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Container, Row, Col } from "../../../reusecore/Layout";
+import Button from "../../../reusecore/Button";
+import ProfileCard from "../../../components/Profile-card";
+import CommunityManagersWrapper from "./styles";
+import { useStaticQuery, graphql } from "gatsby";
+
```
Comment (warunicorn19) [inline src/sections/Community/CommunityManagers/index.js:7]:
```suggestion
import {Link} from "gatsby";
import { IoIosArrowRoundForward } from "react-icons/io";

```

File: src/sections/Community/Meshmates/index.js [hunk 4]
```diff
@@ -127,6 +127,9 @@ const Meshmates=() => {
             </Col>
           </Row>
         </div>
+        <div className="conduct">
+          <h4> Explore about Community Manager at <a href="/community/managers">Layer5 Community Managers</a></h4>
```
Comment (warunicorn19) [inline src/sections/Community/Meshmates/index.js:131]:
```suggestion
          <h4> Get to know about <Link to="/community/managers">Layer5 Community Managers <IoIosArrowRoundForward /></a></h4>
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 3800d8a1d4bc733bd42c5fe19d9fad43e647ac0f at: https://61e743e4492e1500ac708549--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thanks @Chadha93

**Pull request merged**